### PR TITLE
Minor editorials

### PR DESCRIPTION
**Pull request by iherman**
- minor grammatical mistake
- the labels for some of the SVG references were wrong (the section numbers have changed...)

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/did-use-cases/pull/138.html" title="Last updated on Jan 27, 2021, 9:48 AM UTC (1ac312c)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/did-use-cases/138/5d3a81c...1ac312c.html" title="Last updated on Jan 27, 2021, 9:48 AM UTC (1ac312c)">Diff</a>

### CODE REVIEW

**Comment by philarcher** [comment]
Thanks @iherman Actually, I need to move the whole SVG up in the doc so things might change again, but this is helpful. Thank you. I'll merge once things have stabilized.

File: index.html [hunk 1]
```diff
@@ -1691,7 +1691,7 @@ <h4>Background</h4>
...
-reqObjectList.push(new ReqObject("orgMort", "Survives issuing organization mortality", "These identifiers survive the demise of the organization that issued them. They usefulness of these identifiers survive organizations going out of business, being purchased (and potentially losing domain names or root credentials), and even the internal decay of an organization that no longer has the ability to verify the authenticity of records they once issued.", ['sustainability']));
+reqObjectList.push(new ReqObject("orgMort", "Survives issuing organization mortality", "These identifiers survive the demise of the organization that issued them. The usefulness of these identifiers survive organizations going out of business, being purchased (and potentially losing domain names or root credentials), and even the internal decay of an organization that no longer has the ability to verify the authenticity of records they once issued.", ['sustainability']));
```
Comment (TallTed) [inline index.html]:
```suggestion
reqObjectList.push(new ReqObject("orgMort", "Survives issuing organization mortality", "These identifiers survive the demise of the organization that issued them. The usefulness of these identifiers survives organizations going out of business, being purchased (and potentially losing domain names or root credentials), and even the internal decay of an organization that no longer has the ability to verify the authenticity of records they once issued.", ['sustainability']));
```

**Comment by philarcher** [comment]
With all use cases now written and merged, this kind of thing is all that's left to do. I'll take care of this in the coming 48 hours. There are some other new bits to add to the SVG and both @jandrieu and I need to do a full read through, spell check etc. This should also allow us to close Issue #62 that I'm pretty confident we have addressed

**Comment by philarcher** [comment]
The SVG diagram has been updated today with slightly larger text and the last two UCs added. Further changes have been made to the text you cite (correctly) as being grammatically incorrect so that the point is now moot.

Thanks.

**Pull request closed without merging**